frontend/src/routes/Search.tsx

```diff
@@ -98,9 +117,11 @@ const SearchPage: React.FC<Props> = ({ q, outcome }) => {
     }
 
     return <>
-        <Breadcrumbs path={[]} tail={t("search.title", { query: q })} />
+        <Breadcrumbs path={[]} tail={t("search.title", {
+            query: q,
+            hits: outcome.__typename === "SearchResults" ? outcome.items.length : 0,
```

Comment:
The idea of showing the number of results is good, but this is actually misleading as the search endpoint has a limit of how many items to return. MeiliSearch can return the total number of hits. We just need to pipe it through, i.e. also expose it in our API. Specifically, you'd need to add a field `totalHits` to `SearchResults` (the Rust struct). In the four places that create a `SearchResults`, the field needs to be specified of course. We can likely just use `results.estimated_total_hits`. But huff. [Related docs](https://www.meilisearch.com/docs/reference/api/search#exhaustive-and-estimated-total-number-of-search-results). Not sure if we want to introduce pagination at some point to our search page? Ehm... lets decide that later :D Its a bit much for one comment here.